src/frontend/src/app/shared/workers/flow-generator.worker.ts

```diff
@@ -50,12 +63,23 @@ parser.on('opentag', (tag: TagForOptions<{}>) => {
     flowStructure.nodes.push(currentNode);
   } else if (currentNode.type.match(/Exit$/g)) {
     flowStructure.nodes.push(currentNode);
+  } else if (currentNode.type === 'Pipeline') {
+    pipeline = currentNode;
```

Comment:
For what can the pipeline be used? As a repleacement for the firstPipe? Shouldn't is be nice to that remove the firstPipe code?

Reply:
It is necessary in order to insert first pipe at the right position if you have an empty pipeline.

Reply:
But then it could serve a bouble purpose.